cmd/wasm-client/main.go

```diff
@@ -84,14 +84,19 @@ func main() {
 		exit(err)
 	}
 
-	wc := newWasmClient(&cfg)
+	wc, err := newWasmClient(&cfg)
+	if err != nil {
+		exit(err)
+	}
 
 	// Setup JS callbacks.
-	js.Global().Set("wasmClientIsReady", js.FuncOf(wc.IsReady))
-	js.Global().Set("wasmClientConnectServer", js.FuncOf(wc.ConnectServer))
-	js.Global().Set("wasmClientIsConnected", js.FuncOf(wc.IsConnected))
-	js.Global().Set("wasmClientDisconnect", js.FuncOf(wc.Disconnect))
-	js.Global().Set("wasmClientInvokeRPC", js.FuncOf(wc.InvokeRPC))
+	callbacks := js.ValueOf(make(map[string]interface{}))
+	callbacks.Set("wasmClientIsReady", js.FuncOf(wc.IsReady))
+	callbacks.Set("wasmClientConnectServer", js.FuncOf(wc.ConnectServer))
+	callbacks.Set("wasmClientIsConnected", js.FuncOf(wc.IsConnected))
+	callbacks.Set("wasmClientDisconnect", js.FuncOf(wc.Disconnect))
+	callbacks.Set("wasmClientInvokeRPC", js.FuncOf(wc.InvokeRPC))
+	js.Global().Set(cfg.NameSpace, callbacks)
```

Comment:
This is just for our CLI example though right? So actual client code needs to handle this itself right (I guess our npm module in this case).